src/server/server.js

```diff
@@ -218,13 +219,35 @@ SimulationServer.prototype._streamAppHostHtml = function (request, response) {
                 return '<script src="' + scriptSource + '"></script>';
             }).join('');
 
-            // Note we replace "default-src 'self'" with "default-src 'self' ws:" (in Content Security Policy) so that
-            // websocket connections are allowed (this relies on a custom version of send that supports a 'transform' option).
+            // Note we add "connect-src 'self' ws:" and "img-src 'blob:' (in Content Security Policy) so that
+            // websocket connections are allowed and the camera plugin works (this relies on a custom version
+            // of send that supports a 'transform' option).
+            var metaTagRegex = /<\s*meta[^>]*>/;
```

Comment:
I've tested these changes with the cordova mobilespec app, but with a little modification. The cordova mobilespec app analyzes the user agent to identify mobile runtime, and if it matches some of the OS like android, ios etc, it will inject CSP. That is done automatically when one of the scripts is loaded. Since for cordova-simulate with do  not change the user agent yet, I took the CSP configuration and add it manually to the index.html, looking like this:
```
<html>
  <head>
    <!-- meta tag #1 -->
    <meta name="viewport" content="width=device-width,height=device-height,user-scalable=no,initial-scale=1.0" />
    <!-- meta tag #1 -->
    <meta http-equiv="Content-type" content="text/html; charset=utf-8"/>
    <!-- meta tag #3 - CSP -->
    <meta http-equiv="Content-Security-Policy" content="default-src 'self' https://ssl.gstatic.com/accessibility/javascript/android/; connect-src 'self' http://cordova-filetransfer.jitsu.com; media-src 'self' http://cordova.apache.org/downloads/ https://cordova.apache.org/downloads/; frame-src 'self' http://stealbridgesecret.test/ data: gap:; img-src 'self' data: cdvfile:; style-src 'self' 'unsafe-inline'"/>
    <!-- more content here -->
 
  </head>
  <body id="stage" class="theme">
    <!-- more content here -->
  </body>
</html>
```

With the changes in this branch, and also with latest master, replacing CSP while streaming the index.html doesn't work. The regex to match the meta tag only matches the first meta tag, not all of them. But if we change the regex to use the global modifier,
 ```
var metaTagRegex = /<\s*meta[^>]*>/g;
```
it returns all of the matches.